pom.xml

```diff
@@ -46,6 +50,19 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+
+      <dependency>
+        <groupId>io.quarkus.qe</groupId>
+        <artifactId>common-test-utils</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      
+      <dependency>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>kafka</artifactId>
+        <version>${testcontainers.version}</version>
+        <scope>test</scope>
```

Comment:
Testcontainers are not used across all modules, I would prefer to import it in individual module that requires it

Reply:
The module is declared within the dependency management, so this dependency won't be automatically added across all the modules. 
If one module wants to use it, the module needs to add it as a dependency.